tests/foreman/cli/test_activationkey.py

```diff
@@ -1172,7 +1155,7 @@ def test_positive_copy_ak2(self):
         self.assertEqual(result[0], u'Activation key copied')
 
     @tier1
-    def test_negative_copy_ak(self):
+    def test_negative_copy_duplicate_name(self):
```

Comment:
duplicate or same? since there are other tests performing a similar test, we should try to use one or the other.